core/src/main/java/io/kestra/core/models/tasks/NamespaceFiles.java

```diff
@@ -0,0 +1,40 @@
+package io.kestra.core.models.tasks;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import javax.validation.Valid;
+
+@Builder
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Introspected
+public class NamespaceFiles {
+    @Schema(
+        title = "Enabled injections of namespace files"
+    )
+    @PluginProperty
+    @Builder.Default
+    private Boolean enabled = true;
+
+    @Schema(
+        title = "Filter to include only matching glob patterns"
```

Comment:
```suggestion
        title = "A list of filters to include only matching glob patterns"
```
could be more explicit this way

Reply:
could be also: "Filters to include only matching glob patterns" if you don't want to include the info that it's a list 👍